library/control/src/modules/ProductFeatures.rb

```diff
@@ -43,6 +43,9 @@ def main
       # See defaults map below for sample contents
       @features = nil
 
+      # See SetOverlay and ClearOverlay
+      @backup_features = nil
```

Comment:
actually not needed, but if you would like to have it here for documentation purpose, it is fine for me.